rules/windows/lateral_movement_suspicious_cmd_wmi.toml

```diff
@@ -0,0 +1,43 @@
+[metadata]
+creation_date = "2020/10/19"
+ecs_version = ["1.6.0"]
+maturity = "production"
+updated_date = "2020/10/19"
+
+[rule]
+author = ["Elastic"]
+description = """
+Identifies suspicious Cmd execution via Windows Management Instrumentation (WMI) on a remote host. This could be
+indicative of adversary lateral movement.
```

Comment:
```suggestion
Identifies suspicious command execution (cmd) via Windows Management Instrumentation (WMI) on a remote host. This could be
indicative of adversary lateral movement.
```